src/import_organization_csv.py

```diff
@@ -0,0 +1,35 @@
+import os
+import django
+import pandas as pd
+
+# 設置 Django 環境
+#os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'myproject.settings')
+#django.setup()
+
+from Organization.models import WelfareOrganization
+
+# 讀取 CSV 文件
+csv_file_path = 'welfare_organization_data.csv'
```

Comment:
csv_file_path = 'welfare_organization_data.csv'
```suggestion
csv_file_path = 'res/welfare_organization_data.csv'
```